### PR TITLE
fix(boot): fix incorrect comment

### LINKED ISSUE #1756: Maybe some mistakes in comment?
## Description / Steps to reproduce / Feature proposal
In [datasource.booter.ts](https://github.com/strongloop/loopback-next/blob/7e1acfc1e72da89e22421c8e94aaac3ba9083e0d/packages/boot/src/booters/datasource.booter.ts#L49) , the comment seems copyed from repository.ts.
## Current Behavior
The comment is about repository, not datasource
## Expected Behavior
comment about datasource

_See [Reporting Issues](http://loopback.io/doc/en/contrib/Reporting-issues.html) for more tips on writing good issues_

### PR DESCRIPTION
**Pull request by virkt25**
fixes #1756 

## Checklist

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

### CODE REVIEW

File: packages/boot/src/booters/datasource.booter.ts [hunk 1]
```diff
@@ -47,9 +47,9 @@ export class DataSourceBooter extends BaseArtifactBooter {
     await super.load();
 
     /**
-     * If Repository Classes were discovered, we need to make sure RepositoryMixin
+     * If DataSource Classes were discovered, we need to make sure RepositoryMixin
      * was used (so we have `app.repository()`) to perform the binding of a
```
Comment (bajtos) [inline packages/boot/src/booters/datasource.booter.ts]: Actually, I think this should say `app.dataSource()`, not `app.repository()`!

Comment (virkt25) [reply packages/boot/src/booters/datasource.booter.ts]: @bajtos Great catch! Fixed.

File: packages/boot/src/booters/datasource.booter.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Copyright IBM Corp. 2018. All Rights Reserved.
+:x// Copyright IBM Corp. 2018. All Rights Reserved.
```
Comment (Amir-61) [inline packages/boot/src/booters/datasource.booter.ts]: You should remove `:x`?

Comment (virkt25) [reply packages/boot/src/booters/datasource.booter.ts]: 😨 oops

**Pull request merged**